### ISSUE TITLE
"Class 'RealRashid\SweetAlert\Facades' not found"

### ISSUE THREAD

**Issue by amine8ghandi8amine** [realrashid/sweet-alert#8]
Laravel 5.5

**Comment by realrashid** [comment]
resolved in #6

**Comment by realrashid** [comment]
firstly run
``` bash
composer dumpautoload
```
if still not resolve the goto config/app.php aliases array add 
``` php
'Alert' => RealRashid\SweetAlert\Facades\Alert::class,
```
